src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import statusReducer from './categories/categories';
+import booksReducer from './books/books';
+
+// Establish the root reducer
+const rootReducer = combineReducers({
+  booksReducer,
+  statusReducer,
+});
+
+// Set up the store
+const store = configureStore(rootReducer);
+
```

Comment:
- [ ] [Reuired] Configure store accepts a single configuration object that contains a reducer key and some other keys. To configure your store, you need to pass the `rootReducer` variable as a value to the reducer key when using the `configureStore.` 👍 

#### Hint: `const store = configureStore({reducer: rootReducer})`. Kindly make adjustments. 👍 

- [ ] [Optional] `configureStore` automatically calls `combineReducers` for you, so you might not want to do that directly. Your code can look like this:

```javascript
const store = configureStore({
  reducer: {
   books: booksReducer,
   categories: statusReducer
  },
});
```

Reply:
@Gambit142 Thank you very much for the hint!